src/components/PeopleTable.tsx

```diff
@@ -0,0 +1,95 @@
+import { Link, useParams } from 'react-router-dom';
+import { EnrichPerson } from '../types';
+import classNames from 'classnames';
+
+type Props = {
+  people: EnrichPerson[];
+};
+
+export const PeopleTable: React.FC<Props> = props => {
+  const { people } = props;
+  const { personSlug } = useParams();
+
+  const isPersoninList = (slug: string | undefined) => {
+    const result = people.some(person => person.slug === slug);
+
+    return result;
+  };
+
+  return (
+    <table
+      data-cy="peopleTable"
+      className="table is-striped is-hoverable is-narrow is-fullwidth"
+    >
+      <thead>
+        <tr>
+          <th>Name</th>
+          <th>Sex</th>
+          <th>Born</th>
+          <th>Died</th>
+          <th>Mother</th>
+          <th>Father</th>
+        </tr>
+      </thead>
+      <tbody>
+        {people.map(person => (
+          <tr
+            data-cy="person"
+            key={person.slug}
+            className={classNames({
+              'has-background-warning': person.slug === personSlug,
+            })}
+          >
+            <td>
+              <Link
+                to={`../${person.slug}`}
+                className={classNames({
+                  'has-text-danger': person.sex === 'f',
+                })}
+              >
+                {person.name}
+              </Link>
+            </td>
+
+            <td>{person.sex}</td>
+            <td>{person.born}</td>
+            <td>{person.died}</td>
+            <td>
+              {person.motherName ? (
+                <>
+                  {isPersoninList(person.mother?.slug) ? (
```

Comment:
I suggest moving this and similar parts into a separate component, PersonLink, so that we can reuse it wherever we need without repeating the same code. As far as I know, the same thing was established in the task description